client/app/components/ParameterMappingInput.jsx

```diff
@@ -213,7 +214,7 @@ export class ParameterMappingInput extends React.Component {
       <Select
         value={mapping.mapTo}
         onChange={mapTo => this.updateParamMapping({ mapTo })}
-        dropdownMatchSelectWidth={false}>
+        dropdownMatchSelectWidth={getItemOfPercentileLength(existingParamNames, 80)}>
```

Comment:
```suggestion
        dropdownMatchSelectWidth={false}>
```

We can keep this one with `false`, the number of Parameters in a Dashboard shouldn't be big enough to need virtualization.